core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala

```diff
@@ -365,7 +417,7 @@ private[transaction] class TransactionMetadata(val transactionalId: String,
             s"this operation is to remove the transaction metadata from the cache, not to persist the $toState in the log.")
       }
 
-      debug(s"TransactionalId $transactionalId complete transition from $state to $transitMetadata")
+      info(s"TransactionalId $transactionalId complete transition from $state to $transitMetadata")
```

Comment:
Is this intentional? I think logging all transitions might be too verbose.

Reply:
Unintentional, fixed